### PR TITLE
[RHDX-514] Fix text overflow on article table of contents component

### PR DESCRIPTION
**Pull request by alistairmc**
### JIRA Issue Link
https://projects.engineering.redhat.com/browse/RHDX-514

### Verification Process
go to /articles/c-containers/ and check toc text is not overlapping box and that on resize the toc remains the same width until mobile breakptoint 

https://developer-preview-3576.ext.us-west.dc.preprod.paas.redhat.com/che-custom-stacks/
https://developer-preview-3576.ext.us-west.dc.preprod.paas.redhat.com/articles/c-containers/
https://developer-preview-3576.ext.us-west.dc.preprod.paas.redhat.com/articles/use-marketplace-add-cockroachdb-operator-openshift/

### CODE REVIEW

**Comment by alistairmc** [comment]
rebuild this please

**Comment by alistairmc** [comment]
@staceymosier @sjcox-rh this can be reviewed in the PR the test failing is due to download manager in stage not working.

**Comment by alistairmc** [comment]
retest this please

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/_mixins.scss [hunk 1]
```diff
@@ -26,6 +26,10 @@
   }
 }
 
+@mixin resetComponentPadding {
```
Comment (staceymosier) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/_mixins.scss:29]: @alistairmc  Where is this mixin being used?

**Comment by alistairmc** [comment]
rebuild this please

**Pull request merged**